SrMoffat/MyDiary
# Feature#159028245 Serves the POST api/v1/entries endpoint for CREATING an ENTRY resource

### What does the PR do?
Updates the repo with the POST api/v1/entries endpoint for CREATING an ENTRY resource 
The code for the endpoint is on the `ft-add-entry` branch 

### Description of tasks completed
1. POST api/v1/entries endpoint
2. Tests for the POST api/v1/entries endpoint 

### Manual Testing 
1. Clone the repo
2. Run `python manage.py test` to run tests (4 out 7 should pass --> 3 for config, 1 for the endpoint)
3. Run `python manage.py run` to run the application
4. Navigate to the http://127.0.0.1:5000/api/v1/entries endpoint on the address bar 
5. POST data using the format:
        ```{
            'title' : 'Test Title',
            'content' : 'Test Content'
        }```

### Pivotal Tracker Story
#159028245
### Notes
>Please feel free to leave a comment on the PR and on slack
